crates/sc-consensus-subspace-rpc/src/lib.rs

```diff
@@ -855,4 +867,64 @@ where
             pipe_from_stream(pending, mapping_stream).boxed(),
         );
     }
+
+    fn subscribe_filtered_object_mappings(
+        &self,
+        pending: PendingSubscriptionSink,
+        hashes: Vec<Blake3HashHex>,
+    ) {
+        if hashes.len() > MAX_OBJECT_HASHES_PER_SUBSCRIPTION {
+            warn!(
+                "Request hash count ({}) exceed the server limit: {} ",
+                hashes.len(),
+                MAX_OBJECT_HASHES_PER_SUBSCRIPTION
+            );
+
+            let err_fut = pending.reject(ErrorObject::owned(
+                // TODO: is there a defined error code for an internal request limit?
```

Comment:
Was this supposed to be committed?

Reply:
Yep, is there a defined error code for our internal application request limits?

Reply:
I didn't do any research in this direction and do not know off top of my head. Did you check what Substrate maintainers do in this case?

Reply:
Turns out we already have an `impl Into<ErrorObject> for Error`, so passing the error directly uses the correct code for subspace.